coordinator/src/services/translation/incoming/release/release-medication-request.ts

```diff
@@ -0,0 +1,297 @@
+import * as fhir from "../../../../models/fhir/fhir-resources"
+import {
+  DaysSupply,
+  DispensingSitePreference,
+  DosageInstructions,
+  LineItem,
+  LineItemQuantity,
+  Performer,
+  Prescription,
+  PrescriptionEndorsement,
+  PrescriptionTreatmentType,
+  PrescriptionType,
+  ReviewDate
+} from "../../../../models/hl7-v3/hl7-v3-prescriptions"
```

Comment:
At what point do we just import *, can't be much more in this file

Reply:
Yeah I've not looked at imports. I'll switch everything to the annoying but tidy style where we `import * as x` instead

Reply:
Done